gc/base/MemorySubSpaceFlat.cpp

```diff
@@ -459,6 +459,12 @@ MM_MemorySubSpaceFlat::collectorExpand(MM_EnvironmentBase* env, MM_Collector* re
 
 uintptr_t
 MM_MemorySubSpaceFlat::releaseFreeMemoryPages(MM_EnvironmentBase* env)
+{
+	return releaseFreeMemoryPages(env, 0);
+}
+
+uintptr_t
+MM_MemorySubSpaceFlat::releaseFreeMemoryPages(MM_EnvironmentBase* env, uintptr_t includeMemoryType)
```

Comment:
Not needed. The one without params should this work directly (and what it does is correct - just passes the request to its child subspace, which also does not need any parameters).

Reply:
I'm actually only partially correct here. Not needed for Gencon, but it is for Optthruput and Optavgpause, where this would be a top level Subspace (top level child of Space).
To deal with these 2 configuations we can:
- do exactly what you've done - ignore the fact the caller requested release of NEW memory, since we don't know anything about that
- assert that we are called with exact type OLD, but make sure that caller is aware which Configuration we are running and pass appropiate flags.
I think the former is simpler, so you can just leave this as is (or call the variant without parameters - as already said in previous comment, that one must pass the request to its child).